.github/workflows/deploy-prod-mainnet.yml

```diff
@@ -4,13 +4,17 @@ on:
     branches: [main]
 jobs:
   deploy-mainnet:
-    uses: NEARBuilders/bos-workspace/.github/workflows/deploy.yml@main
-    with:
-      bw-legacy: false
-      deploy-env: "mainnet"
-      app-name: "[APP_NAME]"
-      deploy-account-address: "[DEPLOY_ACCOUNT]"
-      signer-account-address: "[SIGNER_ACCOUNT]"
-      signer-public-key: [PUBLIC_KEY]
-    secrets:
-      SIGNER_PRIVATE_KEY: ${{ secrets.SIGNER_PRIVATE_KEY }}
+    name: Deploy components to social.near (mainnet)
+    runs-on: ubuntu-latest
+    environment: treasury-devdao.near
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install dependencies
+        run: npm ci
+      - name: Build components
+        run: npm run bw:build:treasury
+      - name: Navigate to build directory
+        run: cd ./build/treasury-devdao.near
+      - name: Deploy components
+        run: npm run deploy -- '${{ vars.NEAR_SOCIAL_ACCOUNT_ID }}' sign-as '${{ vars.NEAR_SOCIAL_ACCOUNT_ID }}' network-config mainnet sign-with-plaintext-private-key --signer-public-key '${{ vars.NEAR_SOCIAL_ACCOUNT_PUBLIC_KEY }}' --signer-private-key '${{ secrets.NEAR_SOCIAL_ACCOUNT_PRIVATE_KEY }}' send
```

Comment:
there is `npm run deploy`, but I don't see any `deploy` in package.json `scripts`

Reply:
ohh, added